### PR TITLE
doc: describe more about troubleshooting tips

### PR DESCRIPTION
**Pull request by dongsupark**
Describe more about troubleshooting tips, mainly about setting up nodes as well as requirements. Fix typos and change names too.

Closes https://github.com/kinvolk/kube-spawn/issues/81

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ $ sudo GOPATH=$GOPATH CNI_PATH=$GOPATH/bin ./kube-spawn setup --image=coreos --n
 $ sudo GOPATH=$GOPATH CNI_PATH=$GOPATH/bin ./kube-spawn init
 ```
 
+Now that you're up and running, this is how you can start using it.
+
```
Comment (robertgzr) [inline README.md]: I think this still needs something here ^^

File: doc/troubleshooting.md [hunk 2]
```diff
@@ -30,7 +60,7 @@ in another version is totally fine.
 You may try to use master branch from upstream systemd repository, but we
 don't encourage it.
 
-You can pass `kubeadm-nspawn` an alternative `systemd-nspawn` binary by setting the
+You can pass `kube-spawn` an alternative `systemd-nspawn` binary by setting the
```
Comment (robertgzr) [inline doc/troubleshooting.md]: nice job catching these :)

File: README.md (hunk 1 above)
Comment (dongsupark) [inline README.md]: @robertgzr I just changed it simply to `"Now that you're up and running, you can start using it."`

**Comment by dongsupark** [comment]
For better readability, I moved the section of `Deploying to your local cluster` to the next of  `Quickstart`, and added a line `"kubectl get pods"` to the example. Now readers would get better impression of what to do after setting up nodes.
Added `(optional)` to the title of `Running local Kubernetes builds`, as it's not always necessary.
Added a line `"sudo iptables -P FORWARD ACCEPT"` to the example of `SELinux and firewalld`.

File: README.md [hunk 3]
```diff
@@ -49,7 +49,23 @@ $ sudo GOPATH=$GOPATH CNI_PATH=$GOPATH/bin ./kube-spawn setup --image=coreos --n
 $ sudo GOPATH=$GOPATH CNI_PATH=$GOPATH/bin ./kube-spawn init
 ```
 
-## Running local Kubernetes builds
+Now that you're up and running, you can start using it.
+
+## Deploying to your local cluster
+
+`kube-spawn` creates `tmp/` inside the directory you run it in.
```
Comment (blixtra) [inline README.md]: Is we are now creating the `.kube-spawn` directory, why don't we just put that there too?

Comment (dongsupark) [inline README.md]: @blixtra This will be done via https://github.com/kinvolk/kube-spawn/pull/102

Comment (blixtra) [inline README.md]: I guess this needs to be updated now.

Comment (dongsupark) [inline README.md]: @blixtra Done. Rebased and updated the document again.

**Pull request merged**